TESTING.md

```diff
@@ -109,7 +109,14 @@ To run a test file in either directory, `cd` into it before running the tests.
 ```bash
 cd shared
 bundle exec ruby -Itest ./test/path/to/your/test.rb
-``` 
+bundle exec ruby -Itest ./test/test_html_parser.rb
+```
+
+```bash
+cd lib
+bundle exec ruby -Itest ./test/path/to/your/test.rb
+bundle exec ruby -Itest ./test/cdo/aws/test_cloudwatch_logs.rb
```

Comment:
This was probably an accident? 😂

Reply:
No, I did that on purpose because I always forget how to do it :sweat_smile: